src/components/PlantingSites/editor/PlantingSiteEditor.tsx

```diff
@@ -155,13 +163,21 @@ export default function PlantingSiteEditor(props: PlantingSiteEditorProps): JSX.
 
   const onStartOver = () => {
     // TODO: reset data here, confirm with user?
```

Comment:
Should this TODO be removed?

Reply:
good catch, yes will remove ( in a follow up )